package.json

```diff
@@ -16,10 +16,10 @@
     "url": "https://github.com/bae-sh/tiptap-extension-resize-image"
   },
   "scripts": {
-    "build": "yarn clean && yarn build:tsc && yarn build:rollup",
+    "build": "yarn build:tsc && yarn build:rollup",
     "build:tsc": "yarn tsc --declaration --emitDeclarationOnly --declarationDir dist",
     "build:rollup": "rollup -c rollup.config.mjs",
-    "clean": "rm -rf dist esm",
+    "clean": "del /f /q dist esm && rmdir /s /q dist esm",
```

Comment:
Supporting both macOS and Windows seems challenging, so if you prefer, please consider using the forked repository. 🙏